.github/workflows/go-lint.yml

```diff
@@ -11,17 +11,19 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Set up Go
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v5
         with:
           go-version: 1.22
       - name: Go report card
         uses: creekorful/goreportcard-action@v1.0
```

Comment:
Since this GitHub Action has not been updated in 4-5 years and only involves a simple command:
```sh
curl -X POST -F "repo=github.com/$GITHUB_REPOSITORY" https://goreportcard.com/checks
```
we might consider two alternatives:
1. Replace the action with the direct `curl` command and link to the results at: https://goreportcard.com/report/github.com/rilldata/rill
2. Use the `goreportcard` CLI to display code quality metrics directly in the GitHub Action console: https://github.com/gojp/goreportcard#command-line-interface